contents/docs/cdp/batch-exports/databricks.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Databricks destination for batch exports
+sidebar: Docs
+showTitle: true
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+Batch exports can be used to export data to a Databricks Delta table.
+
+> **Note:** The Databricks destination is currently in `beta`. This means the configuration and features are subject to change.
+>
+> You can request access via the [in-app support form](https://us.posthog.com/#panel=support%3Asupport%3Abatch_exports%3Alow%3Atrue).
+
+## Requirements
+
+To create a batch export to Databricks, you need the following:
+
+- A Databricks account with the [Premium plan or above](https://www.databricks.com/product/pricing/platform-addons)
+- A Databricks workspace with Unity Catalog enabled
+- A SQL warehouse
+- A catalog and schema you wish to use
+- A service principal with [OAuth Machine-to-Machine (M2M)](https://docs.databricks.com/aws/en/dev-tools/auth/oauth-m2m) authentication configured. This is the only authentication type we support currently. This service principal must have the appropriate permissions (see below)
+
+## Setup instructions
+
+### 1. SQL warehouse
+
+Batch exports uses SQL warehouses to export data into your Delta tables.
+
+You can use either an existing warehouse, or you can [create a new one](https://docs.databricks.com/aws/en/compute/sql-warehouse/create).
+
+You will need to make a note of the **Server hostname** and **HTTP path** for your warehouse:
+
+1. In Databricks, navigate to **SQL Warehouses**.
+2. Select your SQL warehouse.
+3. Go to the **Connection details** tab.
+4. Copy the **Server hostname** (e.g. `abc123.cloud.databricks.com`).
+5. Copy the **HTTP path** (e.g. `/sql/1.0/warehouses/abc123def456`).
+
+### 2. Set up catalog and schema
+
+1. In your Databricks workspace, create a catalog (or use an existing one) where PostHog will export data.
+2. Create a schema within that catalog (or use an existing one).
```

Comment:
what are the requirements for this schema? I assume it needs to be formed a certain way related to your ph data?